src/mesh/Draco_Mesh.hh

```diff
@@ -133,8 +134,14 @@ public:
   const std::vector<std::vector<double>> &get_node_coord_vec() const {
     return node_coord_vec;
   }
-  const std::vector<unsigned> &get_cell_type() const { return m_cell_type; }
-  const std::vector<unsigned> &get_cell_to_node_linkage() const {
+  const std::vector<unsigned> &get_num_faces_per_cell() const {
```

Comment:
Returning a `const&` to a class member can be a little risky.  If the class goes out of scope the `const&` will be invalid.  Is this really what you want here?

Reply:
That is a good question - the intent is to always have the class in scope, and to not copy this data when accessed, but this does seem risky.  Do you have any recommendations?

Reply:
I would prefer that you return a copy unless the memory cost is truly prohibitive.

Reply:
Will do - just going to make one other tweak before pushing.